File: Js/InsertSort.js
const InsertSort = (l) => {
  const sortedL = [...l];
  for (let i = 1; i < l.length; i++) {
    const temp = sortedL[i];
    let j = i;
    while (j > 0 && sortedL[j - 1] > temp) {
      sortedL[j] = sortedL[j - 1];
      j--;
    }
    sortedL[j] = temp;
  }
  return sortedL;
};

const l = [2, 5, 1, 7, 3, 9, 43, 12, 13, 13, 54, 23, 11];
console.log(InsertSort(l));

File: Python/InsertSort.py
#InsertSort
def InsertSort(l):
    for a in range(1,len(l)):
        #Take last item
        temp=l[a]
        
        #Do new counter
        j=a-1
        
        #Do cycle until temp will be most litle
        while j>=0 and temp<l[j]:
            l[j+1]=l[j]
            j-=1
        
        #Give temp place  
        l[j+1]=temp
    return l

#Create ourList and run our Sort
ourList=[2,5,1,3,5,9,7,8,4,2]
print(InsertSort(ourList))